src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/HelidonServer.java

```diff
@@ -149,5 +158,18 @@ public void adaptMavenModel(Model pomFile, JessieModel model, boolean mainProjec
         pomFile.addProperty("helidonVersion", helidonVersion);
         pomFile.addProperty("mpVersion", mpVersion);
 
+        Parent parent = new Parent();
+        parent.setGroupId("io.helidon.applications");
+        parent.setArtifactId("helidon-mp");
+        parent.setVersion(helidonVersion);
+        parent.setRelativePath("");
+        pomFile.setParent(parent);
+
+        if (!javaVersionForCompile.isEmpty()) {
```

Comment:
This means that the JDK 8 setting on the UI is ignored and will result in an error when mvn compile command is launched using JDK 8.

Reply:
That's true, although we (Helidon) have documented that beginning with our release 2.0.0 Java 11 or later is required. 

I know currently that the user's input for the MP release controls whether Java 11 is offered as an option. Ideally the starter data model and U/I would allow individual implementations to optionally indicate which Java versions they accept, so in this case the Helidon 2.x releases would indicate that only 11 or later are OK. 

I did not want to force that enhancement onto the starter infrastructure so I took this approach, but for sure it would be a better user experience if the tool had this added flexibility.

Reply:
Shall I file a separate issue requesting this new feature in the starter?

Reply:
https://github.com/eclipse/microprofile-starter/issues/344

Reply:
I have updated the PR now that #346 has been merged.